tests/test_files_cli.py

```diff
@@ -0,0 +1,280 @@
+    responses.add(
+        responses.GET,
+        "https://analysispreservation-dev.cern.ch/api/deposits/some-pid",
+        json={
+            'links': {
+                'bucket': 'http://analysispreservation.cern.ch/api/files/bucket-id'
+            }
+        },
+        status=200)
+
+    responses.add(
+        responses.GET,
+        'http://analysispreservation.cern.ch/api/files/bucket-id/test.txt',
+        body=b'Cannot GET /files/bucket-id/test.txt',
+        status=404)
+
+    res = cli_run("files download -p some-pid test.txt")
+
+    assert res.exit_code == 1
+    assert res.stripped_output == 'Cannot GET /files/bucket-id/test.txt'
+
+
+@responses.activate
+def test_files_remove(cli_run):
+    responses.add(
+        responses.GET,
+        "https://analysispreservation-dev.cern.ch/api/deposits/some-pid",
+        json={
+            'links': {
+                'bucket': 'http://analysispreservation.cern.ch/api/files/bucket-id'
+            }
+        },
+        status=200)
+
+    responses.add(
+        responses.DELETE,
+        'http://analysispreservation.cern.ch/api/files/bucket-id/test.txt',
+        body=b'test',
+        status=204)
+
+    res = cli_run("files remove -p some-pid test.txt")
+
+    assert res.exit_code == 0
+    assert res.stripped_output == 'File test.txt removed.'
+
+
+def test_files_remove_no_pid_given(cli_run):
+    res = cli_run("files remove test.txt")
+
+    assert res.exit_code == 2
+    assert "Error: Missing option '--pid' / '-p'." in res.output
+
+
+def test_files_remove_no_filename_given(cli_run):
+    res = cli_run("files remove -p some-pid")
+
+    assert res.exit_code == 2
+    assert "Error: Missing argument 'FILENAME'." in res.output
+
+
+@responses.activate
+def test_files_remove_pid_not_exists(cli_run):
+    responses.add(
+        responses.GET,
+        'https://analysispreservation-dev.cern.ch/api/deposits/some-pid',
+        json={
+            'status': 404,
+            'message': 'PID does not exist.'
+        },
+        status=404)
+
+    res = cli_run("files download -p some-pid test.txt")
+
+    assert res.exit_code == 1
+    assert res.stripped_output == 'PID does not exist.'
+
+
+@responses.activate
+def test_files_remove_file_not_exists(cli_run):
+    responses.add(
+        responses.GET,
+        "https://analysispreservation-dev.cern.ch/api/deposits/some-pid",
+        json={
+            'links': {
+                'bucket': 'http://analysispreservation.cern.ch/api/files/bucket-id'
+            }
+        },
+        status=200)
+
+    responses.add(
+        responses.DELETE,
+        'http://analysispreservation.cern.ch/api/files/bucket-id/test.txt',
+        body=b'Cannot DELETE /files/bucket-id/test.txt',
+        status=404)
+
+    res = cli_run("files remove -p some-pid test.txt")
+
+    assert res.exit_code == 1
+    assert res.stripped_output == 'Cannot DELETE /files/bucket-id/test.txt'
+
+
+@responses.activate
+def test_files_upload_file(cli_run):
```

Comment:
we need also some test to check the case with uploading a directory (that tar was created and sent to the server endpoint correctly)